### ISSUE TITLE
date/clock module calendar css

### ISSUE THREAD

**Issue by littleblack111** [Alexays/Waybar#3686]
as shown in the example of clock's calindar, i can add the color class my self to a element in the module,json if i want color at all.

I do want color, and i have wallust(pywal alternative). so i need css

**Comment by Machenblau** [comment]
I have the same issue, i want to change the colour and add a Border around the current day (and maybe a animation on the day changing)(inside the Calendar Tooltip), while also keeping all colour definitions in one File(ideally a seperate css that is included via `@import`)

i have such a File and would like to reuse it:
```css
:root{
    --col_text:rgba(176, 180, 188,1);
    --col_background:rgba(0,0,0,0.8);

    --col_Shadow_active:rgba(0,0,26,1);
    --col_Shadow_inactive:rgba(0,0,0,0.66);

    --col_border_active_1:rgba(125,2,2,0.66);
    --col_border_active_2:rgba(160,8,9,0.66);

    --col_border_inactive_1:rgba(125,2,2,0.26);
    --col_border_inactive_2:rgba(160,8,9,0.26);

    --col_button_hover:rgba(200,20,20,1);
}
 ```

the Inspector is not Helpful in this case, as the Tooltip dissappears, once the mouse is moved to the Inspector.

**Comment by nickjj** [comment]
Yes, I would really like to be able to style the calendar using CSS classes. Right now they do not work and you're limited to `color=` attributes directly in the `config.jsonc` file.

If you try to add a custom `class=` attribute then Waybar errors out with `... from markup due to error parsing markup: Attribute 'class' is not allowed on the <span> tag on line 1 char 82`.

This is problematic for everyone who wants to have dynamic themes on their system. If it were in CSS, you could have theme specific CSS file that gets symlinked to take effect without needing to modify the `config.jsonc` file itself. This is how the rest of Waybar can be configured.

**Comment by nickjj** [comment]
I took a shot at adding support for this here https://github.com/Alexays/Waybar/pull/4896.
